packages/coverage-c8/src/provider.ts

```diff
@@ -58,7 +58,7 @@ export class C8CoverageProvider implements CoverageProvider {
         if (!map)
           return
 
-        const url = _url.pathToFileURL(file).href
+        const url = _url.pathToFileURL(file).href.replace(/\?(.*)/, '')
```

Comment:
Isn't there already a `pathname` which does not contain the query string?

- https://nodejs.org/api/url.html#urlpathname

```suggestion
        const url = _url.pathToFileURL(file).pathname
```

Reply:
Using `pathname` will return only the path of the URL, without the [origin](https://nodejs.org/api/url.html#urlorigin) part, which is needed for the `html` report (`{ reporter: 'html' }`) to redirect to the proper .html coverage file
I could do a concatenation of `.orign` + `.pathname` ? Looks less coming out of nowhere than the replace